.travis.yml

```diff
@@ -2,6 +2,7 @@
 sudo: false
 language: ruby
 cache: bundler
+dist: trusty
```

Comment:
[This commit](https://github.com/logstash-plugins/logstash-input-github/commit/c4db8fbecc195f3a795c730be223cc02e2d553b3) that was added after your initial commit should fix the build issue without having to downgrade Travis to using `trusty`

Reply:
Ah, thanks for pointing that out!

I've rebased on master, and removed the commit where this change was made.